pkg/service/ioservice_sip.go

```diff
@@ -31,7 +32,7 @@ func (s *IOInfoService) matchSIPTrunk(ctx context.Context, trunkID, calling, cal
 	if err != nil {
 		return nil, err
 	}
-	return sip.MatchTrunk(trunks, "", calling, called)
```

Comment:
@dennwc I am guessing this is fine given that it was passing empty string before. Can you please confirm?

Reply:
Going to merge this. But, please do let me know if this change is incorrect.